integration-tests/bats/log.bats

```diff
@@ -254,14 +265,14 @@ teardown() {
     [[ "$output" =~ "BRANCH1" ]] || false
 
     # Should default to first argument as branch name, second argument as table name (if table exists) if two arguments provided
-    run dolt log myname myname 
+    run dolt log myname -- myname
```

Comment:
Yeah we're no longer supporting table names without the separator. I think @macneale4 and I decided that would be best since it's really hard to resolve revisions otherwise.

Reply:
After discussing with Neil I'm going to push a change to support table names without the separator. Should make us backwards compatible.